src/PrestaShopBundle/Resources/config/services/adapter/country.yml

```
@@ -24,23 +24,26 @@ services:
   PrestaShop\PrestaShop\Adapter\Country\Repository\CountryRepository:
     autowire: true
 
-  prestashop.adapter.country.command_handler.add_country_handler:
+  PrestaShop\PrestaShop\Adapter\Country\CommandHandler\AddCountryHandler:
     public: true
-    class: 'PrestaShop\PrestaShop\Adapter\Country\CommandHandler\AddCountryHandler'
     autowire: true
     tags:
       - { name: 'tactician.handler', command: 'PrestaShop\PrestaShop\Core\Domain\Country\Command\AddCountryCommand' }
 
-  prestashop.adapter.country.command_handler.edit_country_handler:
+  PrestaShop\PrestaShop\Adapter\Country\CommandHandler\DeleteCountryHandler:
+    public: true
```

```suggestion
    public: false
```
unless its used somewhere by $container->get() and is too complicated too do now?